uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/utils/AdbShellUtils.java

```diff
@@ -78,6 +86,7 @@ public static void installPackage(@NonNull final String packageName) {
      * @param flags       the flags to use during installation of the app
      */
     public static void installPackage(@NonNull final String packageName, @NonNull String... flags) {
+        Logger.i(TAG, "Installs the supplied package on the device with the supplied flags..");
```

Comment:
let's log the package name and flags in the log message

Reply:
added.